### ISSUE TITLE
sza and saa missing parameter

### ISSUE THREAD

**Issue by agronomofiorentini** [hectornieto/pyTSEB#51]
Dear Creators,
I have been at https://landtraining2022.esa.int/ and i am trying to use your module but i have several problems.

after running the following command in the terminal inside of the folder

```
python TSEB_local_image_main.py Config_LocalImage.txt
```

with two different methods `TSEB_PT` and `DTD` i have the following output in the console

```
Run pyTSEB with configuration file = Config_LocalImage.txt
Error: missing parameter sza
pyTSEB will not be run due to errors in the input data.
```

```
Run pyTSEB with configuration file = Config_LocalImage.txt
Error: missing parameter saa
pyTSEB will not be run due to errors in the input data.
```

How can i solve this?

**Comment by hectornieto** [comment]
Hi! Thanks for reaching out and raising this issue, 

I think this issue is fixed now in commit [78acde78ce4a4da0a71a48013f3fe119dbf44785](https://github.com/hectornieto/pyTSEB/commit/78acde78ce4a4da0a71a48013f3fe119dbf44785)

You could now optionally include SZA and SAA in the configuration file, otherwise pyTSEB use lat lon fields together with overpass time to computed those two angles

**Comment by agronomofiorentini** [comment]
Yes it is working perfectly now
